dash/backend/package.json

```diff
@@ -39,78 +39,79 @@
     "cli:database:migrate": "node ${CLI_PATH:=./dist/src/cli.js} database:migrate"
   },
   "dependencies": {
-    "@kubernetes/client-node": "^0.18.1",
```

Comment:
Are you intending to be including the package upgrades? I'm just clarifying since the PR title is 'Issue 6775' with no other context

Reply:
When I started this issue i saw main branch contained https://github.com/m9sweeper/m9sweeper/pull/329'r code.
So I started based on issue-7059's code.